### PR TITLE
Fix some typos in markdown docs

### PR DESCRIPTION
**Pull request by JoeWrightss**
1. "ptraceable" to "traceable" in line 5.
2. "preferrable" to "preferable" in line 32.

### CODE REVIEW

**Comment by justincormack** [comment]
"ptraceable" is correct if ugly. Maybe better to rework it to "unable to `ptrace`" or something.

**Comment by JoeWrightss** [comment]
Thank you for guiding me, @justincormack🐯.

**Comment by justincormack** [comment]
Its the other way round, it is "prevent other processes from being able to ptrace it"

**Comment by JoeWrightss** [comment]
I have update it, @justincormack🐯.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## [v0.6.1](https://github.com/docker/notary/releases/tag/v0.6.0) 04/10/2018
 + Fixed bug where CLI requested admin privileges for all metadata operations, including listing targets on a repo [#1315](https://github.com/theupdateframework/notary/pull/1315)
-+ Prevented notary signer from being dumpable or ptraceable in Linux, except in debug mode [#1327](https://github.com/theupdateframework/notary/pull/1327)
++ Prevented notary signer from being dumpable or able to ptrace in Linux, except in debug mode [#1327](https://github.com/theupdateframework/notary/pull/1327)
```
Comment (endophage) [inline CHANGELOG.md]: Per @justincormack's feedback, it should be "Prevented notary signer from being dumpable or being ptraced ..."

**Comment by HuKeping** [comment]
Rebased to master and update per @justincormack's feedback, , waiting for CI.

**Review by HuKeping — APPROVED** [review]

**Pull request merged**